pages/platform/ai/notebook_tuto_10_create_chatbot/guide.en-gb.md

```diff
@@ -0,0 +1,118 @@
+
+### Understand storage concepts
+
+The obvious goal about using object storage and not the local storage of the AI Notebook, is to de-correlate compute and storage, allowing us to stop or delete the notebook while keeping the data safe.
+If you want to know more about data storage concept please fill free to check this : [Create an object container](https://docs.ovh.com/gb/en/storage/object-storage/pcs/create-container/). 
+
+For the chatbot, we will create one object storage bucket. This bucket will be filled over time by our trained model output. The container where the model will be saved don't have to be created. When we will launch our notebook, the container will be automatically created. 
+
+Now, let's start to create and train our chatbot.
+
+### Create and train a chatbot with a Visual Studio Code notebook
+
+ In order to do it you will need to create a vscode notebook. You can attach one volume if you want to save the model created. 
+Don't forget to put the model inside the folder `trained-models` before stopping your notebook. By doing this, your model will be save in your object container even if you delete your notebook. Here is the command to run: 
+
+
+``` console
+ovhai notebook run conda vscode \
+--name vscode-ovh-chatbot \
+--framework-version conda-py39-cuda11.2-v22-4 \
+--volume <model-output-container>@GRA:/workspace/trained-models:RW \
+--cpu 10 \
+--token <token> \
+--label model=rasabotRO \
+-s ~/.ssh/id_rsa.pub
+```
+
+Few explanations here, line by line:
+- It will launch a notebook with the Python Conda framework an VScode editor.
+- With the name "vscode-ovh-chatbot".
+- We specify a version for the conda framework (Python 3.9). 
+- we attach 1 data volume to the notebook, as explained previously.
+- We request 10 CPUs. The more we add, the more performant it is (but more expensive).
+- we add the token previously created.
+- We labelize this notebook with label "rasabotRO".
+- Optional : the last line is the path to your ssh key on your machine. It is important to setup the key here to connect by remote on VScode. If you don't want to, you can connect directly on the web with the token you create before.
+
+> [!primary]
+>
+> Note that the repository git don't have to be added because it is already present when you create an AI Notebook. 
+>
+
+Your notebook is ready, you can connect to him. 
+
+But just before training the notebook, you must install all the dependencies of rasa. So open a terminal and launch this command !
+```console 
+pip install rasa
+```
+
+Once you installed all the dependecies, you can create and train the chatbot with only one command in your terminal. Run at the root of your directory :
+```console
+rasa init
+```
+
+It will create some folders with basic data for rasa chatbot. During the command, you can also train a model and speak with the chatbot at the end. Here is a little example of a conversation with the chatbot. 
```

Comment:
```suggestion
It will create some folders with basic data for rasa chatbot. During the command, you can also train a model and speak with the chatbot at the end. Here is a small example of a conversation with the chatbot. 
```